src/index.php

```diff
@@ -1,2 +1,40 @@
 <?php
 
+namespace BiscuitApp;
+
+require_once 'vendor/autoload.php';
+
+use PDO;
+
+// connecting to biscuit database
+$db = new PDO('mysql:host=db; dbname=biscuits', 'root', 'password');
+$hydrateBiscuitObject = new BiscuitHydrator();
+
+// saving allBiscuitsObject from a variable into the database
+$allBiscuitsObject = $hydrateBiscuitObject->getBiscuitObjects($db);
```

Comment:
the method `getBiscuitObjects` is static - no need to create instance of the hydrator. please change accordingly